cognite/extractorutils/_inner_util.py

```diff
@@ -25,6 +25,45 @@ def _resolve_log_level(level: str) -> int:
     return {"NOTSET": 0, "DEBUG": 10, "INFO": 20, "WARNING": 30, "ERROR": 40, "CRITICAL": 50}[level.upper()]
 
 
+def resolve_log_level_for_httpx(level: str) -> str:
+    return {
+        None: "WARNING",
+        "INFO": "WARNING",
+        "WARNING": "WARNING",
+        "ERROR": "ERROR",
+        "CRITICAL": "CRITICAL",
+    }.get(level, "WARNING")
```

Comment:
We talked about setting the level to debug, I don't see any of that here?

This appears to _increase_ the level of all logging.

Reply:
>We talked about setting the level to debug, I don't see any of that here?

Yea sorry that was my bad. I hadn't pushed that change earlier